examples/quickstart/locustfile.py

```diff
@@ -33,3 +39,71 @@ def classify(self):
         self.client.post("/classify", json=input_data.tolist())
 
     wait_time = between(0.01, 2)
+
+
+class GrpcUser(User):
+    abstract = True
+
+    stub_class = None
+
+    def __init__(self, environment):
+        super().__init__(environment)
+        self.environment = environment
+        self.pb, self.services = import_generated_stubs()
```

Comment:
Small optimization here, I think the generated imports can just be imported at the top level